website/content/en/docs/olm-integration/generation.md

```diff
@@ -152,6 +152,9 @@ bundle: ...
   ...
   operator-sdk bundle validate ./bundle --select-optional name=operatorhub
 ```
+The `operatorhub` validator also allows validation of custom `categories` in CSV [annotations][csv_annotation]. This can be done by
+defining a list of custom categories in a json-encoded file and setting its path as an environment variable named `OPERATOR_BUNDLE_CATEGORIES`. For more
+details refer to [validation of operator bundles][olm_validation].
```

Comment:
To avoid doc drift, something like the following would be better:
```suggestion

Documentation on optional validators:
- [`operatorhub`][operatorhub_validator]
```